tests/test_datetime.py

```diff
@@ -1,6 +1,25 @@
 import unittest
+from datetime import datetime, timezone
+
+from hypothesis import given, settings, strategies as st
+from hypothesis.extra.dateutil import timezones as dateutil_timezones
+
+TIME_ZONES_STRATEGY = st.one_of(
+    st.sampled_from([None, timezone.utc]), dateutil_timezones()
```

Comment:
I actually didn't check, so I'm not sure if `dateutil_timezones()` ever returns `None`. I gather that it does return `dateutil.tz.UTC`, so I'm not sure if this more complicated strategy is worth it just to get `timezone.utc` in there if `dateutil_timezones()` can return `None`.

Reply:
`dateutil_timezones()` can't return None - because it's not a dateutil timezone 😉 - so IMO this more complicated strategy is the right option.